ex2_system/include/system_state/state_task.h

```diff
@@ -20,9 +20,42 @@
 #ifndef EX2_SYSTEM_INCLUDE_SYSTEM_STATE_STATE_TASK_H_
 #define EX2_SYSTEM_INCLUDE_SYSTEM_STATE_STATE_TASK_H_
 
+#include <stdbool.h>
+
+#include "eps.h"
 #include "main/system.h"
 
+enum SAT_state {
+  hw_critical_state = 0,  // EPS will shut OBC down prior to informing it
+  critical_state = 1,     // EPS will shut OBC down prior to informing it
+  safe_state = 2,
+  operational_state = 3
+};
+
+/* It might be a good idea to replace it with SAT_returnState. */

```

Comment:
thoughts? (Please check the .c file for the use case)

Reply:
I think this is fine!